### PR TITLE
Release 0.2.7

### PR DESCRIPTION
**Pull request by TimoKramer**
pom.xml needs the release version before merging.

### CODE REVIEW

**Comment by whilo** [comment]
I think we should be able to merge this now.

File: deps.edn [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{:paths ["src"]
+ :deps {org.clojure/clojure        {:mvn/version "1.10.1"}
+        org.clojure/clojurescript  {:mvn/version "1.10.764"}
+        org.clojure/data.fressian  {:mvn/version "0.2.1"}
+        com.cognitect/transit-clj  {:mvn/version "0.8.319"}
+        com.cognitect/transit-cljs {:mvn/version "0.8.256"}
+        fress                      {:mvn/version "0.3.2"}}
```
Comment (TimoKramer) [inline deps.edn]: This should be 'fress/fress'

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>jar</packaging>
+  <groupId>io.replikativ</groupId>
+  <artifactId>incognito</artifactId>
+  <version>0.2.7-SNAPSHOT</version>
```
Comment (TimoKramer) [inline pom.xml]: This should be 0.2.7

**Comment by whilo** [comment]
@TimoKramer Thanks for pointing the two release issues out. It should be good to go now.

**Pull request merged**